app/assets/js/rtMedia.backbone.js

```diff
@@ -1292,11 +1292,8 @@ jQuery( document ).ready( function( $ ) {
 				 * if user want media and activity_text both require
 				 * By: Yahil
 				 */
-				if ( jQuery.trim( jQuery( "#whats-new" ).val() ) == "" ) {
-					if ( rtmedia_activity_text_with_attachment == 'disable') {
-						$( "#whats-new" ).css( 'color', 'transparent' );
-						$( "#whats-new" ).val( '&nbsp;' );
-					} else {
+				if ( '' === jQuery.trim( jQuery( "#whats-new" ).val() )) {
```

Comment:
Please add single space before closing bracket `)` at the end.

Reply:
resolved it.